posthog-core/src/utils.ts

```diff
@@ -9,27 +9,26 @@ export function removeTrailingSlash(url: string): string {
 }
 
 export interface RetriableOptions {
-  retryCount?: number
-  retryDelay?: number
-  retryCheck?: (err: any) => boolean
+  retryCount: number
+  retryDelay: number
+  retryCheck: (err: any) => boolean
 }
 
-export async function retriable<T>(fn: () => Promise<T>, props: RetriableOptions = {}): Promise<T> {
-  const { retryCount = 3, retryDelay = 5000, retryCheck = () => true } = props
```

Comment:
there was a mismatch of the default value of `retryDelay`, either 3s or 5s depending on the code path, refactored to make it non-optional since `_retryOptions` is always created in the ctor with default values anyway.